routes/songlist-db.js

```diff
@@ -153,8 +160,10 @@ router.route('/:song_id/edit')
         song.year = req.body.year
         song.lang = req.body.lang
         song.copyright = req.body.copyright
-        song.lyric = req.body.lyric
         song.contributor = req.body.contributor
+        var stringArr = req.body.lyric.split(/\r?\n|\//)
```

Comment:
what if the req.body.lyrics is undefined? 
maybe 
`var lyric = (req.body.lyric || '').split(....);
song.lyric = lyric.slice(0);` ?

Also, why do you need to use slice to get the copy?